cast/src/starknet_commands/call.rs

```diff
@@ -6,15 +6,15 @@ use starknet::core::utils::get_selector_from_name;
 use starknet::providers::jsonrpc::HttpTransport;
 use starknet::providers::{JsonRpcClient, Provider};
 
+use crate::starknet_commands::helpers::parse_number;
+
 #[derive(Args)]
 #[command(about = "Call a contract instance on Starknet", long_about = None)]
 pub struct Call {
     /// Address of the called contract (hex)
-    #[clap(short = 'a', long = "contract-address")]
     pub contract_address: String,
 
     /// Name of the contract function to be called
-    #[clap(short = 'f', long = "function-name")]
```

Comment:
Hmm, haven't we decided not to remove those clap configs? 🤔

Reply:
I don't see how is it related? this change is not on master, so this PR is the one that makes the change right? if so, please revert this change as it should be discussed before it is merged

Reply:
as https://github.com/software-mansion/protostar/issues/2117 is closed we can revert [revert claps](https://github.com/software-mansion/protostar/pull/2138/commits/25015334ead2b43450aaa8b86c10061795a508a6)